jplag.frontend-utils/src/main/java/de/jplag/TokenList.java

```diff
@@ -63,6 +65,14 @@ public Token getToken(int index) {
         return tokens.get(index);
     }
 
+    /**
+     * Returns a sequential Stream of the tokens.
+     * @return a stream of the tokens
+     */
+    public final Stream<Token> stream() {
```

Comment:
I would rather change the return type of `allTokens()` to Collection rather than add a stream method. Collections can be directly streamed.

Reply:
Ah, sure, that's a good alternative! Coming right up.